common/app/views/fragments/preRenderJavaScript.scala.html

```diff
@@ -0,0 +1,58 @@
+@()
```

Comment:
It's not really detecting any features, though it is attaching doc class like the others. Would you say it should be grouped with them because of this commonality?